1. Lectura de archivos
- Importa pandas y numpy siguiendo las convenciones.
- Lee la base de datosworldcup2014.csv y asígnela aun objetodf.
- Solicita las primeras 5 observaciones conhead.

In [1]:
import pandas as pd
import numpy as np

df = pd.read_csv('worldcup2014.csv')

df.head(5)

team     continent group  group_pos  cantidad_juegos  \
0       Brazil  southamerica     A          1                3   
1       Mexico  northamerica     A          2                3   
2      Croatia        europe     A          3                3   
3     Cameroon        africa     A          4                3   
4  Netherlands        europe     B          1                3   

   juegos_ganados  juegos_empatados  juegos_perdidos  goles_favor  \
0               2                 1                0            7   
1               2                 1                0            4   
2               1                 0                2            6   
3               0                 0                3            1   
4               3                 0                0           10   

   goles_contra goles_diferencia  puntos  clasificado  
0             2                5       7            1  
1             1                3       7            1  
2             6                0       3            0  
3             9               −8       0            0  
4             3                7       9            1

2. Frecuencia de continent: utilizando value_counts responde lo siguiente:
- ¿Cuál es el continente con una mayor presencia en la muestra?
- ¿Cuál es la probabilidad de elegir un equipo asiático al azar?
- ¿Cuál es la probabilidad de elegir un equipo africano al azar?

In [2]:
df['continent'].value_counts('%')

europe          0.40625
southamerica    0.18750
africa          0.15625
northamerica    0.12500
asia            0.12500
Name: continent, dtype: float64

- El continente con mayor presencia es Europa, con un 40.625% de los equipos siendo de este continente

- La probabilidad de elegir un equipo asiatico es del 12.5%

- La probabilidad de elegir un equipo africano es del 15.625%

3. Generación de un subconjunto de datos: por cada continente, genere un nuevo
objeto que almacene sólo las observaciones del continente.
Tip: Para ello puede utilizar la siguiente sintaxis: df[df['variable'] ==
condicion].


In [ ]:
europe = df[df['continent'] == 'europe']
soutamerica = df[df['continent'] == 'soutamerica']
africa = df[df['continent'] == 'africa']
northamerica = df[df['continent'] == 'northamerica']
asia = df[df['continent'] == 'asia']

4. Probabilidad de clasificación a la siguiente ronda: calcula la probabilidad de que un
país pase a la siguiente ronda (clasifique) por continente.
- ¿Cuál es la probabilidad de que un país asiático pase a la siguiente ronda?
- ¿Cuáles son los dos continentes con la mayor probabilidad de clasificar?
- ¿Cuál es la probabilidad de que un país europeo no clasifique?